### ISSUE TITLE
Action list menu - flat, explicit, informative structure

### ISSUE THREAD

**Issue by erisalke** [bafolts/terraforming-mars#3553]
My ideas about UI improvements. Please consider it as a food for thought and not a request by any means. It's just my vision of most smooth user experience I can imagine. 

### **TL;DR a little bit to the bottom**

**When I play this is my priority checklist for action planning:**
a) check what's up with colonies and politics
b) are there any freebies in global tracks to steal? (24/20/0 heat, 8/16 Venus, oxygent at 9) or any premium map tile (any premium ocean on Tharsis, a South Pole in Hellas or Vulcanos on Elysium
c) what cards I can play now or could possibly play with a little effort (global requirements are one/two steps off or I'm missing a single tag for the card)
d) titles (and awards - if we are in mid to late game) and do I have 8 of any green/red resources to convert to forest/temperature that I should take care off.
x) check the hand before and after each step

In other words I believe the hand is the king. Then the colonies, map, turmoil and prizes are all race, all the rest is secondary - standard projects and 8 magic resources (be careful not to eat a random asteroid, but be even more careful to not open a premium city spot for the next player). Of course everything may vary since Ecoline, Tharsis or Lakefront have their own priorities in all matters...

**Current list of actions goes as follows**
Take your next action
Convert 8 plants to greenery
Convert 8 heat to temperature
Play project card
Trade with a colony tile 
Send a delegate 
Standard projects
Pass for this generation
Sell patents
Undo last action

What I suggest:
1. Get rid of the action menu completely
2. Instead place 4 flat rows of actions
Row one: all available colonies (if I can afford it)
<img width="975" alt="Screenshot 2021-09-01 at 01 19 24" src="https://user-images.githubusercontent.com/15717649/131588005-d30a823e-3661-46b4-88eb-6ba25252b282.png">
Turmoil
<img width="723" alt="Screenshot 2021-09-01 at 01 19 39" src="https://user-images.githubusercontent.com/15717649/131588044-a2f10372-2e4a-44f9-92e2-b6c123681e1d.png">
Hand (with a little twist about what is visible to the user)

Please let the user see all the PLAYABLE cards AND all the "almost" playable cards. What I mean by that? All cards that are in a specific (close) range to be playable should be already visible in the "hand" but somehow marked as disabled / grayed out. By a close range I mean one to two steps off from it's global requirement OR 2 specific delegates/ party ruling OR 1 tag missing to plat.
E.g. "GMO Contract" card is always visible in my hand (greyed out / disabled / last position in the list), since it requires the Greens ruling or 2 Greens' delegates to be played
I want to see grayed out / disabled Heavy Taxiation if I have 1 earth tag (Heavy Taxation requires 2 earths)
I want to see gray out Penguins form 6 oceans (requires 8 to be played)
Same I wish to see (greyed out) Public Celebrations (since it requires just a ruling chairman) or Vote of No Confidence (since it just requires two corruptible party presidents)
Same I wish to see "almost playable" Capitol when there are only 2 or 3 oceans in play (4 is the requirement)
<img width="827" alt="Screenshot 2021-09-01 at 01 19 56" src="https://user-images.githubusercontent.com/15717649/131588087-9b6d1bcc-8e6a-4881-9152-730756b6ba4f.png">
Also, I would move the "sell patents" to the standard projects section, where it belongs, same as "8 heat --> temp" action and "8 plants --> forest "

--------------
### TL;DR; What we have:
<img width="369" alt="Screenshot 2021-09-01 at 02 16 19" src="https://user-images.githubusercontent.com/15717649/131591934-0825e101-69d8-4705-8ea0-614cf441b5d2.png">

What I suggest:
Flat
Explicit
Informative

Colonies
<img width="975" alt="Screenshot 2021-09-01 at 01 19 24" src="https://user-images.githubusercontent.com/15717649/131588005-d30a823e-3661-46b4-88eb-6ba25252b282.png">
Politics
<img width="723" alt="Screenshot 2021-09-01 at 01 19 39" src="https://user-images.githubusercontent.com/15717649/131588044-a2f10372-2e4a-44f9-92e2-b6c123681e1d.png">
Hand
<img width="827" alt="Screenshot 2021-09-01 at 01 19 56" src="https://user-images.githubusercontent.com/15717649/131588087-9b6d1bcc-8e6a-4881-9152-730756b6ba4f.png">
Actions
<img width="380" alt="Screenshot 2021-09-01 at 01 47 17" src="https://user-images.githubusercontent.com/15717649/131589896-4738b647-05be-41e0-8d16-41c7565fbf33.png">
Standards 
<img width="1201" alt="Screenshot 2021-09-01 at 01 48 25" src="https://user-images.githubusercontent.com/15717649/131589993-8a9bdf15-3f0d-4711-b711-5071d4869e6a.png">
Rest
<img width="312" alt="Screenshot 2021-09-01 at 02 02 42" src="https://user-images.githubusercontent.com/15717649/131591060-4f1448a0-39dc-4143-a512-9136256fb31c.png">

Ok sorry for a long post! 
I'm sure this kind of revamp could take a while to implement, yet I believe this is the goal to keep in mind.

Last, you do a great job, the implementation is already fantastic!

**Comment by kberg** [comment]
Today is "make sure every open bug has a comment" day!

This report is over a year old! It's a very thoughtful post, and there's much to like about it.

I don't know if we'll ever rewrite the UI, but this is worth keeping in mind.
